### ISSUE TITLE
Add name to getDevicesInfo

### ISSUE THREAD

**Issue by HakamaObi** [hidglobal/digitalpersona-devices#51]
In normal sdk is posible to get the name of the device like "U are U 4500 FingerPrint Reader" is it possible to add it in the websdk?

**Comment by a-bronx** [comment]
I'll forward the suggestion to WebSdk developers, the `@digitalpersona\devices` library just relays the data received from the SDK. Thank you.

**Comment by praveenaadhtech** [comment]
Yes I am also stuck on this. I have a situation, its detecting laptop default finger print also. without this name information, I couldn't differentiate and connect with correct device.
